owncloud-official.json

```
@@ -21,8 +21,8 @@
                 }
             }
         },
-	    "version": "latest",
-        "description": "Secure file sharing and hosting",
+	"version": "1.1",
+        "description": "Secure file sharing and hosting.<p>Based on the official docker image: <a href='https://hub.docker.com/_/owncloud' target='_blank'>https://hub.docker.com/_/owncloud</a>, available for amd64 architecture only.</p>",
```

```suggestion
        "description": "Secure file sharing and hosting.<p>Based on the official docker image: <a href='https://hub.docker.com/_/owncloud' target='_blank'>https://hub.docker.com/_/owncloud</a>, available for amd64 and arm64 architecture.</p>",
```
@Hooverdan96 
I know this one is due for some work (dericated image) but just correcting while I'm here.